compute/vm-image-spec.yaml

```diff
@@ -15,6 +15,10 @@ commands:
     user: postgres
     sysvInitAction: respawn
     shell: '/usr/local/bin/pgbouncer /etc/pgbouncer.ini'
+  - name: local-proxy
+    user: postgres
+    sysvInitAction: respawn
+    shell: '/usr/local/bin/local-proxy --config-path /etc/localproxy/config.json --pid-path /etc/localproxy/pid --http 0.0.0.0:10432'
```

Comment:
Any chance we can use $XDG_RUNTIME_DIR for the PID path?

https://specifications.freedesktop.org/basedir-spec/latest/

Reply:
I'm open to it, we don't seem to be using it anywhere else currently, so I'll have to check that the var is set for local-proxy and for compute-ctl.

Reply:
Looks to be missing
```
root@compute-...:~# echo $XDG_RUNTIME_DIR

root@compute-...:~#
```

Reply:
(also not provided to compute_ctl env vars)

Reply:
Hmm, that seems like a small issue, but not worth blocking on.